pkg/lib/machine-info.js

```diff
@@ -248,7 +248,7 @@ function processMemory(info) {
             memorySize = cockpit.format(_("$0 GiB"), memorySizeValue / 1024);
         }
 
-        let memoryTechnology = memoryProperty["Memory technology"];
+        let memoryTechnology = memoryProperty["Memory Technology"];
```

Comment:
We didn't dream this up -- back then we collected the actual output and added it to the mock in the test suite. I suppose it was changed in some newer version, but let's please not break older dmidecode versions then - can you please make this case independent?

Reply:
I looked into dmidecode.git and I couldn't find it, as QEMU vm's don't show memory technology I can't really reproduce it here.

Reply:
```
[root@rhel-8-6-127-0-0-2-2201 ~]# strings /usr/sbin/dmidecode  | grep -i techno
Memory Technology
```
Hmmm maybe debian-ism?

But that also reports: (debian-stable)

```
root@debian:~# strings /usr/sbin/dmidecode  | grep -i techno
Memory Technology
```

Reply:
I still think it's fine, but dropped the patch for now.